frontend-react/src/pages/deliveries/Table/DeliveriesTable.test.tsx

```diff
@@ -160,15 +161,16 @@ describe("DeliveriesTableWithNumbered", () => {
             });
 
             describe("TableFilter", () => {
-                test("Clicking on filter invokes the trackAppInsightEvent", () => {
-                    fireEvent.click(screen.getByText("Filter"));
+                test("Clicking on filter invokes the trackAppInsightEvent", async () => {
+                    await userEvent.click(screen.getByText("Filter"));
 
                     expect(mockAppInsights.trackEvent).toBeCalledWith({
                         name: "Daily Data | Table Filter",
                         properties: {
                             tableFilter: {
                                 endRange: "3000-01-01T23:59:59.999Z",
-                                startRange: "2000-01-01T00:00:00.000Z",
+                                // xx/xx/xxxx returns a 06:00:00 vs xxxx-xx-xx's 00:00:00??
```

Comment:
Was this comment intentional?

Reply:
yes, this is confusing behavior

Reply:
Agree - Maybee we need a ticket to investigate and fix this. @stephenkao ?

Reply:
Discrepancy was due to local timezones, this has been fixed in another PR.